cves/CVE-2012-0022.yml

```
@@ -10,27 +10,27 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-189 Numeric Errors
```

Change this entry to only be the #.  Also, even though CWE-189 listed on the CVE entry, none of the related CWEs on that category really fit this vulnerability, CWE-189 mainly deals with actual math errors.  This one may fit better (https://cwe.mitre.org/data/definitions/407.html).
```suggestion
CWE: 189
```